css/app/datepicker.css

```diff
@@ -89,7 +89,16 @@
 	border-left: 0;
 }
 
+.todayHilight {
+	background-color: rgb(255, 255, 170);
+}
+
+#datepicker table tbody tr.uib-weeks td:first-child button {
+	background-color: rgba(240, 240, 240, .9);
+}
+
 #datepicker table tbody tr.uib-weeks td:last-child button {
+	background-color: rgba(240, 240, 240, .9);
```

Comment:
true, my work environment is set in US. This should be changed to work also with an European environment.